GliaWidgets/Sources/ViewModel/EngagementViewModel.swift

```diff
@@ -205,6 +205,29 @@ class EngagementViewModel: CommonEngagementModel {
         )
     }
 
+    func showCriticalErrorAlert(
```

Comment:
There's a SwiftLint error on this file because it's too long.

Reply:
fixed [b90c461](https://github.com/salemove/ios-sdk-widgets/pull/948/commits/b90c46109184312f65ec958cc79f9d6346365c76)

Reply:
fixed trailing space [13950f8](https://github.com/salemove/ios-sdk-widgets/pull/948/commits/13950f84002370ebff124546fee31cbca02e3ff1)